### PR TITLE
Chapter 5: finishing the layout

### PR DESCRIPTION
**Pull request by JOlsen-Neslo**

### CODE REVIEW

File: app/assets/stylesheets/_colours.css.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
```
Comment (bradleymarques) [inline app/assets/stylesheets/_colours.css.scss]: I think we also put new lines at then end of sass/scss files

File: app/views/layouts/_shim.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<!--[if lt IE 9]>
```
Comment (bradleymarques) [inline app/views/layouts/_shim.html.erb]: This looks like it is commented out?  If so, rather remove and put back in when needed

Comment (bradleymarques) [reply app/views/layouts/_shim.html.erb]: Unless I am mistaken and it is actually doing something?

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+    <small>
+        The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+        by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
+    </small>
+    <nav>
+        <ul>
+            <li><%= link_to "About", about_path %></li>
```
Comment (bradleymarques) [inline app/views/layouts/_footer.html.erb]: Can't you use the localizations we have defined?

```suggestion
            <li><%= link_to(I18n.t('about'), about_path) %></li>
```

File: app/views/layouts/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+            <li><%= link_to "Contact", contact_path %></li>
```
Comment (bradleymarques) [inline app/views/layouts/_footer.html.erb]: Similar comment here

File: app/views/layouts/_shim.html.erb (hunk 2 above)
Comment (JOlsen-Neslo) [reply app/views/layouts/_shim.html.erb]: This is html syntax for including an if check

**Pull request merged**